src/components/Footer.tsx

```diff
@@ -137,6 +137,14 @@ export default function Footer() {
               >
                 <Sprite symbol="telegram" width="18px" height="18px" />
               </a>
+              <a
+                href="https://x.com/joinmarket"
```

Comment:
NACK, this account is controlled by Chris Belcher, so is basically dead currently.

Reply:
Oh alright. Mind directing me to the right account?

Reply:
There isn't one for JoinMarket currently.

But guess Jam could have one? @theborakompanioni ?

Reply:
No! Removing all links to walled garden platforms is what should be done :wink: 
https://github.com/joinmarket-webui/jam/issues/750

Reply:
Aaah shaks! haha I was so happy when I made the changes. Anyway, I noticed that the footer was alright and doesn't have any twitter links. We can close this issue. 

Lemme go through the issues tab and see what else I can take on

![Logo](https://github.com/KellyKiiru/jam/blob/master/Screenshot%20from%202024-05-09%2007-18-13.png?raw=true)